lib/common/output.c

```diff
@@ -54,7 +55,7 @@ pcmk__output_new(pcmk__output_t **out, const char *fmt_name, const char *filenam
         return ENOMEM;
     }
 
-    if (filename == NULL || safe_str_eq(filename, "-")) {
+    if (pcmk__str_eq(filename, "-", pcmk__str_null_matches | pcmk__str_casei)) {
```

Comment:
We can drop pcmk__str_casei here :)

Reply:
Fixed.  This is just the kind of thing that happens when you are applying changes automatically.